### PR TITLE
Add Discord Channel

### PR DESCRIPTION
**Pull request by negbie**

### CODE REVIEW

File: internal/alert/provider/discord/send.go [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+import (
+	"github.com/balerter/balerter/internal/alert/message"
+	"github.com/bwmarrin/discordgo"
+)
+
+// Send implements
+func (d *Discord) Send(mes *message.Message) error {
+
+	c, err := discordgo.New("Bot " + d.conf.Token)
```
Comment (negasus) [inline internal/alert/provider/discord/send.go:11]: Maybe, we should create a client in the 'New' func, store it in the Discord sctruct and use in Send func?

File: internal/alert/provider/discord/send_test.go [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+import (
+	"os"
+	"testing"
+
+	"github.com/bwmarrin/discordgo"
+	"github.com/stretchr/testify/require"
+)
+
+func TestSend(t *testing.T) {
+	tk := os.Getenv("DG_TOKEN")
```
Comment (negasus) [inline internal/alert/provider/discord/send_test.go:12]: I'm confused for use Env variables in the test. Can you refactoring this unit test without env variables?

File: internal/alert/provider/discord/send.go (hunk 1 above)
Comment (negbie) [reply internal/alert/provider/discord/send.go:11]: Yip I thought about that too but wasn't sure how to close the session at the end. Mby just let it auto expire? Will do this.

File: internal/alert/provider/discord/send_test.go (hunk 2 above)
Comment (negbie) [reply internal/alert/provider/discord/send_test.go:12]: Yeah it's more a integration test. Sadly I found no mocks for the discord lib. Will think about that.

**Comment by negasus** [comment]
Great, thanks!)

**Pull request merged**